### DISCUSSION TITLE
Colour palette configuration is clumsy and poorly documented.

### DISCUSSION THREAD

**Discussion by EnronEvolved** [ghostty-org/ghostty#3531] [Feature Requests, Ideas] [answered]
I had a bit of trouble figuring out how to configure Ghostty's colour palette this afternoon, because the documentation doesn't make clear that the `palette` option only overrides previous values when `N` also matches.

An example of a valid palette configuration would have obviated this trouble:

```
palette =  0=#000000
palette =  1=#CC0000
palette =  2=#4E9A06
palette =  3=#C4A000
palette =  4=#3465A4
palette =  5=#75507B
palette =  6=#06989A
palette =  7=#FFFFFF
```

It's also not possible to revert a colour back to its default, as it stands, as `N=` is not a valid configuration value.

**Comment by jmckillican** [comment]
ghostty +show-config --default will show default values

**Reply by EnronEvolved** [reply]
This is not equivalent to being able to override a value with its default, as documented.

**Reply by iwa** [reply]
would still be better to find examples in the official docs

**Comment by rhodes-b** [comment, marked as answer]
> It's also not possible to revert a colour back to its default, as it stands, as N= is not a valid configuration value.

this is wrong empty values retain the defaults
https://ghostty.org/docs/config#syntax

The documentation does give an example of how to make a pallete
https://ghostty.org/docs/features/theme#example

also what would it do if you didnt apply `N` override all of them? that seems like a horrible UX decission
